src/main/java/com/setycz/chickens/chicken/EntityChickensChicken.java

```diff
@@ -161,4 +167,47 @@ public void readFromNBT(NBTTagCompound tagCompund) {
         super.readFromNBT(tagCompund);
         setChickenTypeInternal(tagCompund.getInteger(TYPE_NBT));
     }
+    
+    @Override
+    public int getTalkInterval()
+    {
+        return 20 * 60;
+    }
+    @Override
+    protected void playStepSound(BlockPos pos, Block blockIn) {
+    	if(this.rand.nextFloat() > 0.1) {
+    		return;
+    	}
+    	super.playStepSound(pos,  blockIn);
+    }
+    
+    
+    @Override
+    protected Item getDropItem()
+    {
+    	ItemStack dropItemStack = getChickenDescription().createLayItem();
+    	Item dropItem = dropItemStack.getItem();
+    	
+    	if(dropItem == Items.string) {
```

Comment:
I like the idea behind it. However, I would prefer to move this logic in ChickensRegistryItem class to a method called 'createDropItem' to encapsulate the logic there.